### DISCUSSION TITLE
Does AdGuard handle cloaking via SVCB records?

### DISCUSSION THREAD

**Discussion by bt90** [AdguardTeam/AdGuardHome#4158] [Q&A]
See explanation here: https://news.ycombinator.com/item?id=26298339

**Comment by ainar-g** [comment]
Hello.  AdGuard Home responds with an empty response on queries other than the usual one.  For example, to see if e.g. `HTTPS` records are blocked, you can add `cloudflare.com` to your blocklist and compare the results of:

```sh
dig 'cloudflare.com' @8.8.8.8 IN TYPE65
```

With:

```sh
dig 'cloudflare.com' "@${YOUR_AGH_ADDR}" IN TYPE65
```

The first will probably give you the data, while the second, have no answer records.
